hartshorn-config/src/main/java/org/dockbox/hartshorn/config/ConfigurationProviders.java

```diff
@@ -16,34 +16,24 @@
 
 package org.dockbox.hartshorn.config;
 
+import jakarta.inject.Singleton;
 import org.dockbox.hartshorn.application.ApplicationPropertyHolder;
 import org.dockbox.hartshorn.application.context.ApplicationContext;
 import org.dockbox.hartshorn.component.Service;
 import org.dockbox.hartshorn.component.condition.RequiresActivator;
 import org.dockbox.hartshorn.component.processing.Binds;
 import org.dockbox.hartshorn.config.annotations.Configuration;
 import org.dockbox.hartshorn.config.annotations.UseConfigurations;
-import org.dockbox.hartshorn.config.properties.ConfigurationObjectPostProcessor;
 import org.dockbox.hartshorn.config.properties.PropertyHolder;
 import org.dockbox.hartshorn.config.properties.StandardPropertyHolder;
 import org.dockbox.hartshorn.config.properties.StandardURIConfigProcessor;
 import org.dockbox.hartshorn.config.properties.URIConfigProcessor;
 
-import jakarta.inject.Singleton;
-
 @Service
 @RequiresActivator(UseConfigurations.class)
 @Configuration({"application", "classpath:application"})
 public class ConfigurationProviders {
 
-    /**
-     * Registers the default implementation of the property holder before any other (standard)
```

Comment:
Reason for removal: this comment targeted a manual order configuration that has since been removed. Before the dependency graph changes in 0.5.0 there was no order checking, so this was required at the time.